drizzle-kit/src/cli/commands/pgPushUtils.ts

```diff
@@ -56,7 +56,21 @@ function tableNameWithSchemaFrom(
 	return concatSchemaAndTableName(newSchemaName, newTableName);
 }
 
-export const pgSuggestions = async (db: DB, statements: JsonStatement[]) => {
+type SelectResolverOutput = {
+	data: {
+		index: number;
+		value: string;
+	};
+	status: 'submitted';
```

Comment:
same comment on other pr but we dont need this field anymore, just keep the index + value

Reply:
removed and exported 👍 we also needed to add a `question` field that we can display next to the options